### ISSUE TITLE
New Associate Member Application: Hello Identity Co-op

### ISSUE THREAD

**Issue by hythloda** [ossf/Governance-Committee#29]
What is your name?<strong>  Dick Hardt</strong>

What is your email?<strong>  dick.hardt@hello.coop</strong>

What is your member company?<strong>  Hello Identity Co-op</strong>

Identify which category their organization falls under:<strong>  Non-profit</strong>

Link to your website:<strong>  https://hello.coop/ </strong>

What is your organizational mission statement:<strong>  Give users control over their identity</strong>

What drives your interest in joining the OpenSSF?<strong>  Give OSS developers control over their identity</strong>

What are your organization’s aspirations for contributing to the OpenSSF, and how do you anticipate that your membership will bolster the OpenSSF's growth and support?<strong>  We hope to share our insights regarding identity and security, to improve the trust amongst OSS developers, and to improve the trust in OSS by others.</strong>

Could you summarize your organization’s contributions to OpenSSF? <strong>  We are offering a free identity provider for OSS developers to use with Sigstore, and to the Linux Foundation.</strong>

Please include any contributions made to OpenSSF or other OpenSSF projects and open-source projects developed using any OpenSSF dependencies.<strong>  We have added the OpenID Connect device flow to the Hellō Wallet, and have added the Sigstore as an application with the default 'sigstore' identifier. The Hellō Wallet is ready to be used with cosign once https://issuer.hello.coop is added as a provider.</strong>

How many developers do you expect to have contribute to OpenSSF projects in the next 6-12 months? Are there other roles such as researchers, analysts or any other positions that you plan on contributing?<strong>  We are offering to operate a developer identity provider, and to add identity provider features as needed for the Sigstore project.</strong>

How do you currently leverage any OpenSSF resources in your organization?<strong>  We plan on using Sigstore in our OSS projects.</strong>

Do you have signing authority for your entire institution? If no, who does?<strong>  Yes</strong>

Do you agree to follow the OpenSSF Code of Conduct<strong>  Yes</strong>

### **1. Organizational Information/Alignment:**

**Organizational Mission Alignment:**

* Does the organization's mission statement align with the goals of OpenSSF, such as promoting open source security, enhancing software supply chain integrity, or contributing to cybersecurity education?
    - [x] Yes
    - [ ] No

**Non-Profit, Government, or Academic Status:**

* Is the organization a recognized non-profit, government agency, or academic institution?
    - [x] Yes
    - [ ] No

**Brand Alignment and Reputation:**

* Is the organization in good standing within its community and the broader open source ecosystem, with a reputation that aligns with OpenSSF's values and brand?
    - [x] Yes
    - [ ] No

### **2. Commitment to Open Source Security and Contribution:**

**Commitment to Contribution:**

* Has the organization demonstrated a clear interest in actively contributing to the OpenSSF community through development, research, or other relevant activities?
    - [x] Yes
    - [ ] No
* Can the organization commit to contributing a specified minimum number of developers, researchers, analysts, or other professionals to OpenSSF projects within the next 6-12 months?
    - [x] Yes
    - [ ] No

**Commitment to Open Source Security:**

* Has the organization previously contributed to OpenSSF or other open-source projects?
    - [x] Yes
    - [ ] No
 - [ ] If not, do they plan on contributing to the OpenSSF?
* Does the organization maintain or contribute to open-source projects that use OpenSSF dependencies or tools?
    - [x] Yes
    - [ ] No

**Commitment to Open Source Security:**

* Does the organization have a history or a nascent plan to promote, improve, or contribute to open source security beyond its participation in OpenSSF?
    - [x] Yes
    - [ ] No

**Utilization of OpenSSF Resources:**

* Does the organization currently leverage OpenSSF resources, tools, or frameworks within its operations or projects?
    - [x] Yes
    - [ ] No

**Educational and Community Engagement:**
* Does the organization engage in educational activities, community outreach, or other efforts to spread awareness about open source security?
    - [x] Yes
    - [ ] No

### **3. Compliance and Ethics:**
**Open Source Licensing and Compliance:**
* Does the organization adhere to open source licensing standards and demonstrate compliance with open source security best practices?
    - [x] Yes
    - [ ] No

**No Conflict of Interest:**
* Can the organization certify that its membership and contributions to OpenSSF will not pose a conflict of interest with the foundation's objectives and policies?
    - [x] Yes
    - [ ] No

**Comment by Danajoyluck** [comment]
@hythloda  adding Dick's linkedin profile here for people who are interested in knowing more about his contribution and impact to OSS work (ActiveState) and identity work (OpenID Foundation, OAuth 2.0)  
https://www.linkedin.com/in/dickhardt/

**Comment by sevansdell** [comment]
I support this. Looking forward to seeing how Hello Identity Coop's identity contributions are brought forward within our TIs. Please encourage TAC engagement if needed while getting alignment and integrated with TIs.

**Comment by Danajoyluck** [comment]
@sevansdell Do you know if there is a way to do the voting on Hello's associate membership sooner rather than later?  Are things like this able to get voted on asynchronously.  I'm asking because if the GC does not vote until May 30th this means they will likely miss the next Associate Member Town Hall.   I see that the next Associate member Town Hall is also May 30th and then the next one is not until July.  I'm just thinking about the member experience of being on hold for such a long time.  Will they still be motivated to contribute after this amount of time?  I would think the May 30th Associate Town Hall would be an opportune time for them to introduce themselves as new Associate members, assuming all goes well with the approval process.  Unsure about this but just flagging for consideration.

**Comment by sevansdell** [comment]
Dana this is the first time GC has used an asynchronous process to vote and we didn’t have advance notice this was coming. To expedite I’d recommend reaching out to GC voting members individually and amplifying your ask and the timeline. Good luck!!

Get Outlook for iOS<https://aka.ms/o0ukef>


Internal Use - Confidential

________________________________

**Comment by SecurityCRob** [comment]
This is an interesting application.  I'm curious what groups Dick & Hello would see themselves participating in amongst our TIs?

**Comment by Naomi-Wash** [comment]
After careful review, the staff decided to approve the membership application.
